app/frontend/README.md

```diff
@@ -1,45 +1,110 @@
-# Vue Skeleton Frontend
+# Common Service Get Token
 
-This is the Vue Skeleton frontend. It implements a Vue frontend with Keycloak authentication support.
+This tool will automate the process for getting setup to use common services published on [https://apistore.nrs.gov.bc.ca/store/](https://apistore.nrs.gov.bc.ca/store/).
```

Comment:
Same as above

Reply:
I've updated the app and frontend readmes to align with the top-level blurbs.